docs/spfx/build-a-hello-world-web-part.md

```diff
@@ -1,171 +1,160 @@
-# Build a Hello World SharePoint client-side web part
+# Build your first SharePoint client-side web part (Hello world part 1)
 
 >**Note:** The SharePoint Framework is currently in Preview, and is subject to change based on customer feedback.  While we’re in preview, SharePoint Framework web parts are not supported for use in production environments.
 
-## Tutorial 1 Overview
-This tutorial will guide you in building a simple client-side web part using the SharePoint client-side development tools. 
+Client-side web parts are client-side components that run inside the context of a SharePoint page. They are different from server-side web parts that you can already build, package and deploy to SharePoint. Server-side web parts are only available to on-premise installations of SharePoint and cannot be deployed to SharePoint online (Office 365). Client-side web parts can be deployed to SharePoint online, and you can also use modern JavaScript tools and libraries to build them.
```

Comment:
Use the first and last sentences of this paragraph. SharePoint Online, SharePoint on-premises.